### PR TITLE
Clarifies text and projects on the Projects page

### PR DESCRIPTION
**Pull request by GBKS**
See issue #162 for context.

This cleans the page up a lot. We need to finalize the list of projects to include.

### CODE REVIEW

File: projects.md [hunk 1]
```diff
@@ -113,25 +92,46 @@ https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrat
 
 # Projects
 
-Here we highlight some (not all) Bitcoin projects we care about, for a few reasons:
+We don’t just chat about design and create resources for designers, we are also deeply involved with projects to power up Bitcoiners through good design. If you are interested in helping out, check the links for each project, ask directly in the communities of those projects, or reach out in our [Slack]({{ site.slack_invite_url }}).
```
Comment (pavlenex) [inline projects.md:95]: Not a big fan of the copy. However, I am lacking coffee so can't make a sensible suggestion either.
What the paragraph should say is:
- Why do we collaborate with the projects
- How does the collaboration happen
- How can projects get involved 
- How can community members help out

Comment (GBKS) [reply projects.md]: I revised the copy, let me know what you think now.

File: projects.md [hunk 2]
```diff
@@ -113,25 +92,46 @@ https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrat
 
 # Projects
 
-Here we highlight some (not all) Bitcoin projects we care about, for a few reasons:
+We don’t just chat about design, we also actively work on improving design in the Bitcoin ecosystem. Projects launched by community members often focus on creating design resources. With collaborations, we try to help other Bitcoin projects improve the design and user experience of their products. For more details, see our [Collaboration](https://github.com/BitcoinDesign/Meta/blob/master/Collaboration.md) and [Project](https://github.com/BitcoinDesign/Meta/blob/master/Projects.md) documents. If you are interested in getting involved, reach out directly to the projects below, or reach out in our [Slack]({{ site.slack_invite_url }}).
```
Comment (pavlenex) [inline projects.md:95]:
```suggestion
We don't just chat about design. We also actively improve design in the Bitcoin ecosystem. Projects launched by community members often focus on creating design resources. With collaborations, we try to help other Bitcoin projects improve their products' design and user experience. For more details, see our [Collaboration](https://github.com/BitcoinDesign/Meta/blob/master/Collaboration.md) and [Project](https://github.com/BitcoinDesign/Meta/blob/master/Projects.md) documents. If you are interested in getting involved, reach out directly to the projects below, or reach out in our [Slack]({{ site.slack_invite_url }}).
```

File: projects.md (hunk 1 above)
Comment (pavlenex) [reply projects.md]: Way better, I've suggested minor simplification, let me know if you're okay with the suggestion [here](https://github.com/BitcoinDesign/Guide/pull/179#discussion_r594960698)

**Review by pavlenex — APPROVED** [review]
LGTM 🚀

**Pull request merged**